checkpointing/tensorflow/README.md

```diff
@@ -0,0 +1,21 @@
+## Checkpointing in Tensorflow with conda environment and GPU support
+The example uses a pre-configured conda environment to demonstrate how one might implement model checkpointing in Tensorflow.
+
+The Tensorflow example downloads the MNIST dataset and trains an neural net on it for 20 epochs, making a "checkpoint" every fifth epoch. In the event of a runtime interruption, these checkpoints are returned to the user and can be used to resume training, preventing the loss of already-completed work.
+
+The five files include:
+- `checkpoint.sub` ## used to submit the job on HTCondor
+- `run.sh` ## the executable called by `checkpoint.sub`
+- `tf_checkpointing.py` ## the python program with an implementation of checkpointing
+- `tf_checkpointing.tar.gz` ## the conda environment used by HTCondor for tensorflow dependencies -- this is hosted on a squid server and is not included in the repo.
+- `environment.yml` ## An environment file for creating the conda env used in this example
+
+### Usage
+- log into the HTC system.
+- clone this repository: `git clone https://github.com/CHTC/template-GPUs`
+- `cd` into this folder: `cd template-GPUs/checkpointing/conda/tensorflow`
+- because conda environments tend to be large, a Squid caching server is used to host the environment. You can access the environment .tar.gz file from the submit node at /squid/gpu-examples. For more information about using Squid, please review the CHTC guide:
+[Large File Availability Via Squid](https://chtc.cs.wisc.edu/uw-research-computing/file-avail-squid) guide.
+- submit the sample job: `condor_submit checkpoint.sub`
+- upon completion, HTCondor will return a zipped model file, `model.tar.gz`, along with checkpoint files `checkpoint.h5` (checkpointed model) and `checkpoint.txt` (which epoch to resume training on)
```

Comment:
Not to be pedantic, but maybe make these into proper sentences, i.e. capitalize first letter, add ending punctuation. 🤓